standard/template/sections/clause_7_unified_data_model.adoc

```diff
@@ -15,25 +15,25 @@ The unified data model incorporates and extends the following established specif
   - **GDAL geotransform metadata**, used to express affine transformations and interpolation characteristics.
   - **SpatioTemporal Asset Catalog (STAC)** metadata elements for resource discovery and cataloguing (Collection and Item constructs).
 
-The unified model is format-agnostic and describes the abstract structure of resources independently of the physical encoding. It does not redefine the semantics of the CDM or CF conventions, but introduces integration and extension points required to support tiled multiscale data, geospatial referencing, and metadata for discovery.
+The Unified Data Model is format-agnostic and describes the abstract structure of resources independently of the physical encoding. It does not redefine the semantics of the CDM or CF conventions, but introduces integration and extension points required to support tiled multiscale data, geospatial referencing, and metadata for discovery.
 
-This clause specifies the logical composition of the unified model, the external standards it leverages, and the conformance points that facilitate harmonised implementation within the GeoZarr framework.
+This clause specifies the logical composition of the Unified Data Model, the external standards it leverages, and the conformance points that facilitate harmonised implementation within the GeoZarr framework.
 
 === Foundational Model and Standards Reuse
 
 GeoZarr adopts established data model concepts because Zarr itself provides only array storage without semantic interpretation. The Unidata Common Data Model (CDM) provides the conceptual framework for understanding dimensions, variables, and attributes, while CF Conventions provide standardized metadata semantics. This reuse ensures compatibility with existing scientific software while avoiding reinvention of proven concepts.
 
 ==== Common Data Model (CDM)
 
-The CDM defines a generalised schema for representing array-based scientific datasets. The following constructs are reused directly within the unified model:
+The CDM defines a generalised schema for representing array-based scientific datasets. The following constructs are reused directly within the Unified Data Model:
 
 - **Dimensions** – Integer-valued, named axes that define the extents of data variables.
 - **Coordinate Variables** – Variables that supply coordinate values along dimensions, establishing spatial or temporal context.
 - **Data Variables** – Multidimensional arrays representing observed or simulated phenomena, associated with dimensions and coordinate variables.
 - **Attributes** – Key-value metadata elements used to describe variables and datasets semantically.
```

Comment:
the CDM uses a particular type system for attributes that is not a 1:1 match for Zarr's attributes type system.

Reply:
Do you have a suggestion for describing that here?

Reply:
That's what i added so far
- Clarified that CDM concepts are adapted for Zarr's JSON type system
- Acknowledged differences while preserving semantic compatibility